spec/support/feature_spec_helper.rb

```diff
@@ -0,0 +1,89 @@
+module FeatureSpecHelper
+
+  RSpec.configure do |config|
+    # Configure Rspec to include the module in feature specs:
+    config.include FeatureSpecHelper, type: :feature
+  end
+
+  #   =====================   LOG IN   =====================   #
+  #   ======================================================   #
+
+  def fill_out_credentials
```

Comment:
The instance variables `@user`, `@user2` are used in a few methods in the module. For the specs to use these methods, they must use the same variables for their test data. This coupling can be removed by not using `@instance` variables. Instead, consider replacing them with method arguments. This will results in more reusable helper methods.

Here's how this would change some of the methods:

```ruby
# Added user argument
def fill_login_credentials(user)
  within 'form' do
    fill_in( 'Email', with: user.email )
    fill_in( 'Password', with: user.password )
    click_on 'Log in'
  end
end

def log_out
  click_link 'Log out'
end

# Added user argument, and user2_login can
# now be removed.
def user_login(user)
  click_log_in
  fill_login_credentials(user)
end
```